haas/cli.py

```diff
@@ -175,16 +205,16 @@ def list_projects():
     do_get(url)
 
 @cmd
-def project_add_user(project, user):
+def user_add_project(user, project):
```

Comment:
I did update the names in the cli to match. I feel pretty strongly about the http semantics being somewhat coherent, and if we're going to namespace the user stuff, we *can't* have them be calls against projects. Let's just leave it as is.